src/Library/demos/Box/main.css

```diff
@@ -0,0 +1,7 @@
+#card {
+  padding: 24px;
```

Comment:
Since the `.card` CSS class has a shadow, you should add a bit of margin here (or padding to the parent) to prevent clipping the shadow.